Makefile

```diff
@@ -56,3 +59,15 @@ proto/proto.pb.o:
 .PHONY: proto-clean
 proto-clean:
 	rm -rf proto/proto.pb.o proto/*.pb.*
+
+.PHONY: test
```

Comment:
For consistency, the test target should be named test-run and the test-binary target should be named test.

Reply:
@Quik95 .PHONY also needs to be changed. And `test-binary` should be `test`. I think that changes don't need new commit if before commits provide that stuff - similar too documentation commit. Personalny I would squash them to one new commit.

Reply:
I improved this stuff because these features are crucial to the project and need to be merged quickly.